### ISSUE TITLE
Add 'Audio' to 'Media' section of forms

### ISSUE THREAD

**Issue by jesicarson** [participedia/frontend#885]
Form field should be for hyperlinks to sites like soundcloud. (same as video allows for links to yooutube, etc.)

Do we want to allow file upload? Thinking no. 

cc @scottofletcher, ref: https://github.com/participedia/frontend/issues/834#issuecomment-410302725

**Comment by scottofletcher** [comment]
@jesicarson no, I don't think we want to allow audio file uploads. I don't think we need a separate media field for audio either - people can use the existing 'Links' field. 

RE #834: my request to add 'Audio' wasn't for the media section, it was for the 'Content Type' field of the Teaching & Learning Resource entries. See pg 2 of [this doc](https://docs.google.com/document/d/1zJaeQG-EuWQXL8RjIT0XjfOZvCCSfPS217BwIEVP2AM/edit).

**Comment by jesicarson** [comment]
Re having a separate audio field: Guess it depends if we want to support an audio player (like the video player for videos). I thought that would be cool, as with my work a lot of my content could be crafted into podcasts. But it might not be something that many people will use. 

Re #834: I know, but it reminded me that it might be cool to have an audio field on all forms, that is if its not too hard to implement. And if we had it on case/method/org forms, it would make sense to have on teaching resources also.

**Comment by scottofletcher** [comment]
@jesicarson Oh that's fine, I'm down for having a separate audio field! I just thought that would be making more work for you guys. But if it's just a link (and not an upload), that would be great (ties in nicely with #834 since, if the content type is audio, there needs to be a way to attach it)
